tests/cbtest/script/test_uzfs.sh

```diff
@@ -0,0 +1,1361 @@
+#!/bin/bash
+#
+#  (C) Copyright 2017 CloudByte, Inc.
```

Comment:
we need to change the license header

Reply:
ok. Should I add openebs license header?